docs-src/go.mod

```diff
@@ -1,3 +1,5 @@
 module github.com/cobaltspeech/sdk-cubic/docs-src
 
-go 1.13
+go 1.15
+
+require github.com/cobaltspeech/docdock v0.0.0-20201021030030-709475da43cc // indirect
```

Comment:
have you run `go mod tidy`? This may be the result of `go install`ing a binary from this package. If this is actually necessary, ignore me :)

Reply:
oh is this a hugo thing? You can see I really might not know what I'm talking about here.

Reply:
yes, this is how hugo modules work--it allows us to bring in the theme without copying it into the repo.